pkg/spec/types.go

```diff
@@ -142,6 +143,14 @@ type SecretMod struct {
 	meta_v1.ObjectMeta `json:",inline"`
 }
 
+// ImageStreamSpec defines OpenShift ImageStream Object
+// kedgeSpec: io.kedge.ImageStreamSpec
```

Comment:
Make it `ImageStreamSpecPod`, since it does not collide with the `ImageStreamSpec` in Openshift

Reply:
did you really mean `ImageStreamSpecPod`? Or maybe `ImageStreamMod`?

What would be problem with that?
We have other `kedgeSpec` comments colliding with Kubernetes names. 
```
// kedgeSpec: io.kedge.IngressSpec
// kedgeSpec: io.kedge.ConfigMap
// kedgeSpec: io.kedge.IngressSpec
// kedgeSpec: io.kedge.ServiceSpec
```

Reply:
The problem is that when the generation is happening in later stage using `openapi2jsonschema` files are created using the spec name, if it collides one file can override the other file.

Reply:
Shouldn't we then fix everything?

Reply:
Now that I think of it. We would ideally want the controller types name to not collide. So if you look at the file [deploymentspecmod json schema](https://github.com/kedgeproject/json-schema/blob/master/schema/deploymentspecmod.json) it has definition of service, configmap, etc. All the fields will be present in the controller's json schema.

So tl;dr don't collide the names of the controller types, rest won't matter.